app/models/results/web_response_parser.rb

```diff
@@ -4,6 +4,7 @@ module Results
   # Builds (does not save) an answer tree based on answer data in a web response.
   class WebResponseParser
     TOP_LEVEL_PARAMS = %i[
+      type
```

Comment:
Changes to the web response parser are wrapped up in 8646; they are for updating web responses (that work prompted 8707 refactor of response factory).